scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/dataspaces/InputAccessMode.java

```diff
@@ -52,10 +52,28 @@
     TransferFromInputSpace("transferFromInputSpace"),
     /** Transfer files from OUTPUT space to LOCAL space */
     TransferFromOutputSpace("transferFromOutputSpace"),
-    /** GLOBAL to LOCAL */
+    /**
+     * Transfer files from GLOBAL space to LOCAL space
+     */
     TransferFromGlobalSpace("transferFromGlobalSpace"),
-    /** USER to LOCAL */
+    /** Transfer files from USER space to LOCAL space */
     TransferFromUserSpace("transferFromUserSpace"),
+    /**
+     * cache files from INPUT space to CACHE space
+     */
+    CacheFromInputSpace("cacheFromInputSpace"),
+    /**
+     * cache files from OUTPUT space to CACHE space
+     */
+    CacheFromOutputSpace("cacheFromOutputSpace"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00115)